src/main/java/hudson/plugins/active_directory/ActiveDirectoryDomain.java

```diff
@@ -266,6 +272,17 @@ public ListBoxModel doFillTlsConfigurationItems() {
             return model;
         }
 
+        /**
+         * Displays an error message if the provided password is less than 14 characters
+         * while in FIPS mode. This message is triggered when the bindPassword field loses focus.
+         */
+        public FormValidation doCheckBindPassword(@QueryParameter String bindPassword) {
+            if(FIPS140.useCompliantAlgorithms() && StringUtils.length(bindPassword) < 14) {
```

Comment:
```suggestion
            Jenkins.get().checkPermission(Jenkins.ADMINISTER);
            if(FIPS140.useCompliantAlgorithms() && StringUtils.length(bindPassword) < 14) {
```

Reply:
No, please. Check MANAGE isntead to prevent a permissions error in case someone with MANAGE enters the page
```suggestion
            Jenkins.get().checkPermission(Jenkins.MANAGE);
            if(FIPS140.useCompliantAlgorithms() && StringUtils.length(bindPassword) < 14) {
```

Reply:
btw, MANAGE not supported in this plugin: https://github.com/jenkinsci/active-directory-plugin/pull/201/files#r1780770012

Reply:
checking permission should be done for expensive check or checks including sending credentials (i.e possible leaks).
here the code is just checking length of a password so not sure it's really mandatory.

Reply:
we can mark this as resolved right ?as we will not be checking permission

Reply:
Well, that's a security warn, I agree it's not that important here, so if code can be merged with that ok for me

Reply:
correct it's a warning which can be safely ignored